### ISSUE TITLE
dynamic import incorrectly minified

### ISSUE THREAD

**Issue by nitsky** [evanw/esbuild#43]
I am observing that the dynamic import function is being incorrectly minified.

```js
// test.js
(async () => {
  await import('./x.js')
})()
```

Run: `esbuild --minify test.js`

Expected output:
```
(async()=>{await import("./x.js")})();
```

Actual output:
```
(async()=>{await a("./x.js")})();
```

**Comment by nitsky** [comment]
With just `--minify-whitespace`, `import` gets rewritten as `import2`.

**Comment by evanw** [comment]
Thanks so much for the report! It turns out I accidentally already fixed this in 32b7b2cb0398bca090abd479f7aa61abf61d1564, but I haven't pushed a new version yet. And also I'm clearly missing test coverage on this. I'll get to both of those soon.
